src/components/Observations/ObservationsTable.vue

```diff
@@ -0,0 +1,311 @@
+          </b-form-group>
+          <b-form-group id="input-group-instrument-type" label="Instrument Type" label-for="input-instrument-type">
+            <b-form-select
+              id="input-instrument-type"
+              v-model="queryParams.instrument_type"
+              :options="formattedFilterOptions.instrument_type"
+              multiple
+            />
+          </b-form-group>
+          <b-form-group id="input-group-configuration-type" label="Configuration Type" label-for="input-configuration-type">
+            <b-form-select
+              id="input-configuration-type"
+              v-model="queryParams.configuration_type"
+              :options="formattedFilterOptions.configuration_type"
+              multiple
+            />
+          </b-form-group>
+          <b-form-group id="input-group-ordering" label="Ordering" label-for="input-ordering">
+            <b-form-select id="input-ordering" v-model="queryParams.ordering" :options="formattedFilterOptions.ordering" />
+          </b-form-group>
+          <b-button-group>
+            <b-button type="submit" variant="outline-primary" :disabled="isBusy">
+              <span>Filter</span>
+            </b-button>
+            <b-button type="reset" variant="outline-danger" :disabled="isBusy">
+              <span>Reset</span>
+            </b-button>
+          </b-button-group>
+        </b-form>
+      </template>
+      <template v-else>
+        <div class="text-center">
+          <i class="fa fa-spin fa-spinner" />
+        </div>
+      </template>
+    </b-col>
+  </b-row>
+</template>
+<script>
+import $ from 'jquery';
+import Pagination from '@/components/Util/Pagination.vue';
+import { TextDisplay } from '@/components/Util';
+import { formatDate } from '@/util';
+import { paginationAndFilteringMixin } from '@/mixins/paginationMixins.js';
+
+export default {
+  name: 'ObservationsTable',
+  components: {
+    Pagination,
+    TextDisplay
+  },
+  filters: { formatDate },
+  mixins: [paginationAndFilteringMixin],
+  props: {
+    observationPortalApiBaseUrl: {
+      type: String,
+      required: true
+    },
+    // `observationDetailLink` is a function that takes an observation ID as input and returns an object describing the
+    // link to the proposal page, either { "href": ... } for a URL or { "to": ... } for a vue router target.
+    // If supplied, the observation ID displayed for each row in the table will be a link.
+    observationDetailLink: {
+      type: Function,
+      required: false,
+      default: () => {} // By default, we don't generate a link.
+    },
+    // `requestgroupLink` is a function that takes a requestgroup ID as input and returns an object describing the
+    // link to the proposal page, either { "href": ... } for a URL or { "to": ... } for a vue router target.
+    // If supplied, the requestgroup ID displayed for each row in the table will be a link.
+    requestgroupLink: {
```

Comment:
This should be a link to the request, not the requestgroup. A requestgroup is a group of one or more requests, while a request is what is actually scheduled into an observation. So an observation is certainly associated with a requestgroup, but we link to the request here to specifically show which request this observation has been scheduled from.

Reply:
Thanks for the clarification!